index.html

```diff
@@ -101,39 +47,39 @@
     </style>
 
   </head>
-  <body>
+  <body data-cite="html media-source">
     <section id="abstract">
-      This specification defines a <a def-id="mse-spec"></a> [[MEDIA-SOURCE]] byte stream format specification based on the ISO Base Media
-      File Format.
+      <p>This specification defines a [[[MEDIA-SOURCE]]] [[MEDIA-SOURCE]] byte stream format specification based on the ISO Base Media
+      File Format [[ISOBMFF]].</p>
     </section>
 
     <section id="sotd">
       <p>The working group maintains <a href="https://github.com/w3c/mse-byte-stream-format-isobmff/issues">a list of all bug reports that the editors have not yet tried to address</a>;
-      there may also be related open bugs in the [[MEDIA-SOURCE]] repository.</p>
+      there may also be related open bugs in the <a href="https://github.com/w3c/media-source">GitHub repository</a>.</p>
       <p>Implementors should be aware that this specification is not stable. <strong>Implementors who are not taking part in the discussions are likely to find the specification changing out from under them in incompatible ways.</strong> Vendors interested in implementing this specification before it eventually reaches the Candidate Recommendation stage should track the <a href="https://github.com/w3c/media-source">GitHub repository</a> and take part in the discussions.</p>
     </section>
 
     <section id="introduction">
       <h2>Introduction</h2>
-      <p>This specification defines segment formats for implementations of <a def-id="mse-spec"></a> [[!MEDIA-SOURCE]] that choose to support the ISO Base Media File Format
-  <a def-id="iso-14496-12"></a> [[!ISOBMFF]].</p> It defines the MIME-type parameters used to signal codecs, and provides
-      the necessary format specific definitions for <a def-id="init-segments"></a>, <a def-id="media-segments"></a>, and <a def-id="random-access-points"></a> required by
-      the <a def-id="byte-stream-formats-section"></a> of the Media Source Extensions spec.</p>
+      <p>This specification defines segment formats for implementations of [[[MEDIA-SOURCE]]] [[!MEDIA-SOURCE]] that choose to support the ISO Base Media File Format [[!ISOBMFF]].</p>
+      <p>It defines the MIME-type parameters used to signal codecs, and provides
+      the necessary format specific definitions for [=initialization segments=], [=media segments=], and [=random access point=] required by
+      the <a data-cite="media-source#byte-stream-formats">Byte Stream Formats section</a> of the Media Source Extensions specification.</p>
```

Comment:
Maybe: 
```suggestion
      the <a data-cite="media-source-2#byte-stream-formats">Byte Stream Formats section</a> of the [[[media-source-2]]] specification.</p>
```

Reply:
If we start using `-2`, we should probably do it consistently throughout the document.

The use of `-2` makes me realize that we may want to adjust Specref for Media Source Extensions: [`[[media-source]]`](https://www.specref.org/?q=media-source) links to https://www.w3.org/TR/media-source/ pretending it's the Rec, but the versionless URL actually redirects to version 2. The correct Rec URL would be https://www.w3.org/TR/media-source-1/

Alternatively, we could create a `media-source-1` entry in Specref, make the `media-source` entry an alias of level 2, as done for /TR), and continue using `[[media-source]]` to target the latest version automatically.

Reply:
I made the change in Specref. `[[media-source]]` is now an alias of level 2 (and `[[media-source-1]` can now be used if needed to target level 1.